fixtures/eurolite/led-bar-6-qcl-rgbw.json

```diff
@@ -0,0 +1,378 @@
+        },
+        {
+          "dmxRange": [120, 255],
+          "type": "Generic",
+          "comment": "/Applications/QLC+.app/Contents/MacOS/../Resources/Gobos/Others/rainbow.png Sound mode via DMX"
+        }
+      ]
+    },
+    "Auto Program Speed": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "comment": "DMX Auto program speed, increasing"
+      }
+    },
+    "Color Presets": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [10, 19],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [20, 29],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [30, 39],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [40, 49],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffff"]
+        },
+        {
+          "dmxRange": [50, 59],
+          "type": "ColorPreset",
+          "comment": "Green yellow",
+          "colors": ["#d4fb79"]
+        },
+        {
+          "dmxRange": [60, 69],
+          "type": "ColorPreset",
+          "comment": "Cyan",
+          "colors": ["#00fdff"]
+        },
+        {
+          "dmxRange": [70, 79],
+          "type": "ColorPreset",
+          "comment": "Magenta",
+          "colors": ["#ff40ff"]
+        },
+        {
+          "dmxRange": [80, 89],
+          "type": "ColorPreset",
+          "comment": "Yellow",
+          "colors": ["#fffb00"]
+        },
+        {
+          "dmxRange": [90, 99],
+          "type": "ColorPreset",
+          "comment": "Turquoise",
+          "colors": ["#00fdff"]
+        },
+        {
+          "dmxRange": [100, 109],
+          "type": "ColorPreset",
+          "comment": "Magenta",
+          "colors": ["#ff40ff"]
+        },
+        {
+          "dmxRange": [110, 119],
+          "type": "ColorPreset",
+          "comment": "Orange",
+          "colors": ["#ff9300"]
+        },
+        {
+          "dmxRange": [120, 129],
+          "type": "ColorPreset",
+          "comment": "Light Green",
+          "colors": ["#73fa79"]
+        },
+        {
+          "dmxRange": [130, 139],
+          "type": "ColorPreset",
+          "comment": "Magenta",
+          "colors": ["#ff40ff"]
+        },
+        {
+          "dmxRange": [140, 149],
+          "type": "ColorPreset",
+          "comment": "Pink",
+          "colors": ["#8cff8c"]
+        },
+        {
+          "dmxRange": [150, 159],
+          "type": "ColorPreset",
+          "comment": "Light Green",
```

Comment:
Light Green is mentioned twice in the manual. Is this a mistake or is it slightly different colors?

Reply:
Yes, as you've spotted, some colours are repeated in the 'Color presets' chart. E.g. Magenta is C07, C10 and C13; Light Green is C12 and C15. 
I believe this is an intentional feature to group colours that are likey to be switched between to make the transition less jarring. E.g, switching Light Green to Light Blue could be done with C15 -> C16 instead of having to jump from C12 -> C16.
I've not tested it on the light (I only took delivery yesterday) but I would imagine they are the same colours.

Reply:
Will do! :) Thank you!

Reply:
Thank you very much for your effort to fix the points from the checklist :blush: